File: tools/llm-bot/src/main/java/org/finos/springbot/tool/llm/LLMBotApp.java
package org.finos.springbot.tool.llm;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class LLMBotApp {
	
	public static void main(String[] args) {
		SpringApplication.run(new Class[] { LLMBotApp.class }, args);

	}
}

File: tools/llm-bot/src/main/kotlin/org/finos/springbot/tool/llm/LLMBotApp.kt
package org.finos.springbot.tool.llm

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

@SpringBootApplication class LLMBotApp

fun main(args: Array<String>) {
    SpringApplication.run(LLMBotApp::class.java, *args)
}
